src/main/java/org/fundacion/jala/converter/service/javacompiler/JavaCompiler.java

```diff
@@ -5,49 +5,55 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
+ *
+ * @author Jessicka Moya Andrade
  */
 package org.fundacion.jala.converter.service.javacompiler;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.fundacion.jala.converter.models.parameter.JavaParameter;
-
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
 
+/**
+ * This class compiles a java file.
+ */
 public class JavaCompiler {
-    BufferedReader bufferedReader;
+    private BufferedReader bufferedReader;
     private JavaParameter javaParameter;
     private static final Logger LOGGER = LogManager.getLogger();
+
     /**
-     * @param newJavaParameter for all the parameters needed for Java Compiler
-     * @return the result of execution in console
+     * The method compiles a java file.
```

Comment:
I think it would be better "Compiles a java file" instead of "The method compiles a java file"

Reply:
I agree with you, I have already changed to "Compiles a java file".